src/microbe_stage/MicrobeCheatMenu.cs

```diff
@@ -0,0 +1,25 @@
+﻿/// <summary>
+///   Handles the microbe cheat menu
+/// </summary>
+public class MicrobeCheatMenu : CheatMenu
+{
+    public void InfCompounds_toggled(bool button_pressed)
```

Comment:
Godot callbacks should follow normal naming convention and not the godot snake_case convention.

Reply:
Also couldn't these methods be put into the base class? Named like `SetInfiniteCompounds(bool newValue)` that way each separate menu type would just need some logic in the Godot tscn to setup it properly.

Reply:
I've seperated the different menus to get smaller, clearer and more seperated classes. Having seperate classes follows the single-responsibility principle better.

Reply:
I think it would make sense for the base class to provide that functionality. The child classes could have buttons or something which need to set multiple values, so the set helpers would also reduce code there. I think overall it reduces code duplication to provide set methods for all the properties in CheatManager in the base CheatMenu class.

Reply:
I still don't get it.
The `SetInfiniteMP` method, for example, only applies to the editor. It would make sense to split the file into multiple modular and simple classes. This boosts readability and leads to smaller classes

Reply:
In the future we'll have a bunch of editors: multicellular editor, tech editor etc. Each of those it makes sense to have an unlimited building cheat. Meaning that all of those cheat menu classes would need to reimplement the infinite building method. Instead if the infiniteMP (or a renamed version) is in the base class, that reduces the amount of overall code needed and maintenance if for example we ever need to update how the cheat manager is told to enable a cheat.